Podfile.lock

```diff
@@ -880,9 +880,9 @@ SPEC CHECKSUMS:
   UIDeviceIdentifier: e6a801d25f4f178de5bdf475ffe29050d0148176
   WordPress-Aztec-iOS: 7d11d598f14c82c727c08b56bd35fbeb7dafb504
   WordPress-Editor-iOS: 9eb9f12f21a5209cb837908d81ffe1e31cb27345
-  WordPressAuthenticator: 0a2a3699c6ddaabf4a5615f43729170f2bb3031a
+  WordPressAuthenticator: fc22569ef113e728823418120014725a7fdee8df
   WordPressKit: a161c49306369cfa22c648866cee5aba16d7cbac
-  WordPressShared: 8e59bc8cec256f54a7c4cc6c94911adc2a9a65d2
+  WordPressShared: 0aa459e5257a77184db87805a998f447443c9706
   WordPressUI: c5be816f6c7b3392224ac21de9e521e89fa108ac
   WPMediaPicker: 0d45dfd7b3c5651c5236ffd48c1b0b2f60a2d5d2
   wpxmlrpc: 68db063041e85d186db21f674adf08d9c70627fd
```

Comment:
Notice no `Podfile` change because I run `bundle exec pod update`.

That's a questionable approach. It resulted in WordPressKit not updating to the latest version, 7.1.0, because the `Podfile` specifies it as `~> 7.0.0-beta`.

It's not an issue in the context of this code freeze because the changes in 7.1.0 are used in a PR that hasn't been merged into `trunk` yet. But it did make me question whether I should revert to manually checking the various libraries instead of relying only on the `Podfile` specification.

Reply:
It's pretty strange to see CocoaPods picking up the podspec in custom spec repo or trunk randomly, which _feels_ like a bug to me.

Reply:
Sorry, I meant to post this comment for [this line above](https://github.com/wordpress-mobile/WordPress-iOS/pull/20462/files#diff-e5b768ab2330ee29074f2eeadddd406f19e73b14673eb5f9b5ed6b5b77f4e363L626).

Reply:
> It's pretty strange to see CocoaPods picking up the podspec in custom spec repo or trunk randomly, which feels like a bug to me.

What I think happened is that last time I/someone run `pod update` for WordPressShared the pod was only available in our specs repo. For reference, as I'm not sure if it's documented anywhere, we have that custom repo to bypass CocoaPods sometimes taking a while to make new pods available.

Reply:
> What I think happened is that last time I/someone run pod update for WordPressShared the pod was only available in our specs repo. 

Yeah, this is indeed the most likely scenario that would explain that.

> For reference, as I'm not sure if it's documented anywhere, we have that custom repo to bypass CocoaPods sometimes taking a while to make new pods available.

See paaHJt-1SO-p2 for more details